src/api/Neovim.ts

```diff
@@ -161,7 +161,7 @@ export class Neovim extends BaseApi {
   }
 
   /** Call Atomic calls */
-  callAtomic(calls: Array<string>): Promise<[Array<any>, boolean]> {
+  callAtomic(calls: Array<any>): Promise<[Array<any>, boolean]> {
```

Comment:
Would a more specific type be `VimValue`?

Reply:
I think that would probably be better.